exotica_python/src/pyexotica.cpp

```diff
@@ -496,7 +496,8 @@ PYBIND11_MODULE(_pyexotica, module)
     setup.def_static("get_dynamics_solvers", &Setup::GetDynamicsSolvers, "Returns a list of available dynamics solvers plug-ins.");
     setup.def_static("create_solver", [](const Initializer& init) { return Setup::CreateSolver(init); }, py::return_value_policy::take_ownership);    // "Creates an instance of the solver identified by name parameter.", py::arg("solverType"), py::arg("prependExoticaNamespace"));
     setup.def_static("create_problem", [](const Initializer& init) { return Setup::CreateProblem(init); }, py::return_value_policy::take_ownership);  // "Creates an instance of the problem identified by name parameter.", py::arg("problemType"), py::arg("prependExoticaNamespace"));
-    setup.def_static("create_map", [](const Initializer& init) { return Setup::CreateMap(init); }, py::return_value_policy::take_ownership);          // "Creates an instance of the task map identified by name parameter.", py::arg("taskmapType"), py::arg("prependExoticaNamespace"));
+    setup.def_static("create_scene", [](const Initializer& init) { return Setup::CreateScene(init); }, py::return_value_policy::take_ownership);
+    setup.def_static("create_map", [](const Initializer& init) { return Setup::CreateMap(init); }, py::return_value_policy::take_ownership);  // "Creates an instance of the task map identified by name parameter.", py::arg("taskmapType"), py::arg("prependExoticaNamespace"));
```

Comment:
I didn't expose `Setup::CreateMap` on purpose. It gives access to creating a map that will need to the scene which is done outside the initializer. Maps created like this won't work without several other steps that are not documented. It's best to create a problem and get the map from there.

Reply:
The `create_map` binding was there before. It's just highlighted in the diff because the comment that has been shifted by clang-format.

Reply:
Are you actively using it? And in what way? It is a lot of complexity that can irritate users, and it won't work without a scene. So if it's not used by you, let's remove it being exposed.

Reply:
To be precise, `create_map` / `createMap` was added with the first commit that introduced the python wrapper: https://github.com/ipab-slmc/exotica/commit/3e482ea74ee64e34817fb6e5c64147cbf9ff1366 :-)

I am never using `create_map`. If it is never used outside I can also just remove it in this PR. If `Setup::CreateMap` is not intended to be used as part of public API, maybe we should just make it private (or remove it if it is not used at all).

Reply:
Let's remove it then from the public API. There won't be a viable use case.

Reply:
`Setup::CreateMap` was only used by `PlanningProblem` once. I declared it therefore as friend class.